python/python/glide/async_commands/core.py

```diff
@@ -2754,6 +2755,55 @@ async def zdiffstore(self, destination: str, keys: List[str]) -> int:
             ),
         )
 
+    async def bzmpop(
+        self,
+        keys: List[str],
+        modifier: ScoreModifier,
+        timeout: float,
+        count: Optional[int] = None,
+    ) -> Optional[List[Union[str, Dict[str, float]]]]:
+        """
+        Blocks the connection until it pops and returns a member-score pair from the first non-empty sorted set, with
+        the given keys being checked in the order they are provided. The optional `count` argument can be used to
```

Comment:
Do you want to emphasize `keys` as `` `keys` `` too?

Reply:
The existing docs usually do not emphasize plural arguments, for example "keys", "elements", or "members". I'm not exactly sure why. For example Sho asked you to remove the angle quotes around elements [here](https://github.com/aws/glide-for-redis/pull/1315#discussion_r1573656610). The only commands that have angle quotes for "keys" currently are zdiffstore and exists, the other commands don't. So maybe leave as is? Idk, what do you think?

Reply:
Just asking. I'm got tired of that, anything is good for me.